edpop-record-ontology.ttl

```diff
@@ -0,0 +1,140 @@
+# (e.g., Köln or Keulen for the city of Cologne).
+
+# My proposal would be to use these properties that can be extracted from the original record
+# in a deterministic way. That means: only string literals, and also links to other records
+# if they are defined in the original catalogues (such as links to CT Thesaurus).
+# Users will be able to enrich, correct and further normalize this data with annotations.
+# Annotations may also be used, perhaps, for automatic non-deterministic normalizations,
+# such as the place of publication as a link to Wikidata.
+
+edpoprec:title a rdf:property ;
+               skos:prefLabel "Title"@en ;
+               rdfs:domain edpoprec:BibliographicalRecord ;
+               rdfs:range xsd:string .
+
+edpoprec:alternativeTitle a rdf:property ;
+                            skos:prefLabel "Alternative title"@en ;
+                            rdfs:domain edpoprec:BibliographicalRecord ;
+                            rdfs:range xsd:string .
+
+# Or: contributor? And how to define the role of the contributor? Perhaps a separate
+# class is better. The catalogues generally describe the roles of the contributors quite
+# well (author, translator, illustrator, etc.)
+edpoprec:author a rdf:property ;
+                skos:prefLabel "Author"@en ;
+		rdfs:domain edpoprec:BibliographicalRecord ;
+		rdfs:range xsd:string .
+
+edpoprec:publisherOrPrinter a rdf:property ;
+                            skos:prefLabel "Publisher or printer"@en ;
+			    rdfs:domain edpoprec:BibliographicalRecord ;
+			    rdfs:range xsd:string .
+
+edpoprec:placeOfPublication a rdf:property ;
+                            skos:prefLabel "Place of publication"@en ;
+			    rdfs:domain edpoprec:BibliographicalRecord ;
+			    rdfs:range xsd:string .
+
+# Dating has xsd:string as its range, because the date is very often not a simple year
+# in the catalogues but can be anything, like [1650] for an estimation or a range like
+# 1800-1914.
+edpoprec:dating a rdf:property ;
+                skos:prefLabel "Dating"@en ;
+		rdfs:domain edpoprec:BibliographicalRecord ;
+		rdfs:range xsd:string .
+
+# For language, not sure which class to use. There are various IRI collections, of which
+# ISO 639-3 (e.g. https://iso639-3.sil.org/code/nld) and Glottolog
+# (e.g. https://glottolog.org/resource/languoid/id/mode1257) seem most attractive,
+# or perhaps Wikidata. Or just the ISO 639-3 code as a literal?
+edpoprec:language a rdf:property ;
+                  skos:prefLabel "Language"@en ;
+		  rdfs:domain edpoprec:BibliographicalRecord .
+
+edpoprec:format a rdf:property ;
+                skos:prefLabel "Format"@en ;
+		skos:description "Format or physical description" ;
+		rdfs:domain edpoprec:BibliographicalRecord ;
+		rdfs:range xsd:string .
+
+# Leaving out for now: genre (a lot of variation across catalogues, but should nevertheless
+# quickly be visible for VRE users), illustrations (same problem) and notes/description.
```

Comment:
And all properties specific to biographical records. But I can imagine you wanted to see the comments on the initial set of properties, first.